sound/soc/sof/sof-priv.h

```diff
@@ -444,6 +444,9 @@ struct sof_ipc_pcm_ops;
  * @fw_loader:	Pointer to Firmware Loader ops
  * @fw_tracing:	Pointer to Firmware tracing ops
  *
+ * @init:	Optional pointer for IPC related initialization
+ * @fini:	Optional pointer for IPC related cleanup
```

Comment:
I personally have have no idea what 'fini' would stand for. We have no precedent for using 'fini'. can we find a more usual name for this callback? 

If we use the ASoC convention, we have init/exit.